packages/cli/src/commands/bundle.ts

```diff
@@ -98,6 +99,7 @@ export async function handleBundle(
         ref: path,
         dereference: argv.dereferenced,
         removeUnusedComponents,
+        keepUrlRefs: argv['keep-url-references'],
```

Comment:
Two questions I have: should it be `keep-url-references` or `keep-url-refs`? 

What if someone should want to keep other refs such as file refs? Will it require another option? Or should the option not be boolean?

Reply:
What the bundle will it be if we keep file $refs? :) 
Regarding the name, I'm ok with the current one, it is clear enough for me.

Reply:
Okay, that makes sense. I wonder though about references vs refs in the option name.